kubernetes_platform/python/kfp/kubernetes/__init__.py

```diff
@@ -12,26 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '1.1.0'
+__version__ = "1.1.0"
```

Comment:
Can we revert the quote style changes? [This pre-commit hook](https://github.com/kubeflow/pipelines/blob/1fcc68121cd030bd5f8301bf965ec969f170ad77/.pre-commit-config.yaml#L15), if used, will later override this.